includes/view/show-heygov-settings.php

```diff
@@ -91,6 +115,29 @@
 	</form>
 
 	<?php if ($heygov_id) : ?>
+		<div class="heygov-api-key">
+			<form action="<?php echo esc_url($_SERVER['REQUEST_URI']); ?>" method="post">
+
+				<table class="form-table">
+					<tr>
+						<th><label for="heygov_api_key">HeyGov API Key</label></th>
+						<td>
+							<input type="text" name="heygov_api_key" class="regular-text" id="heygov_api_key" value="<?php echo esc_attr($heygov_api_key ?: ''); ?>" />
```

Comment:
@SanjaBudic can you add a pattern for input too? we know the api key always starts with `sk_` so this one should work `<input pattern="sk_[a-zA-Z0-9]{20,29}" ...`